### ISSUE TITLE
Running Bundle Install for Github-Pages: Syntax error

### ISSUE THREAD

**Issue by mccallum-sgd** [ParsePlatform/Docs#507]
```
$ ruby --version
ruby 2.4.3p205 (2017-12-14 revision 61247) [x64-mingw32]
```
My Gemfile:
```
source 'https://rubygems.org'
gem "minima", "~> 2.0"
gem 'github-pages', group :jekyll_plugins
```
___
Running `bundle install` from the directory containing my Gemfile, I get this syntax error:
```
$ bundle install

[!] There was an error parsing `Gemfile`: syntax error, unexpected tSYMBEG, expecting keyword_do or
'{' or '(' - gem 'github-pages', group :jekyll_plugins
                                          ^. Bundler cannot continue.

# from C:/pathToRepo/website/Gemfile:3                                                           
#  -------------------------------------------                                                                    
#  gem 'minima', '~> 2.0'                                                                                          
>  gem 'github-pages', group :jekyll_plugins                                                                       
#  source 'https://rubygems.org'                                                                                   
#  ------------------------------------------- 
```
I've tried it with and without the theme config (`gem 'minima', '~> 2.0'`) line suggested on [StackOverflow](https://stackoverflow.com/a/47736011/4036475).
___
I haven't found anyone else having this problem, any ideas?

**Comment by flovilmart** [comment]
CAN you try with ruby 2.3?

**Comment by mccallum-sgd** [comment]
I did, unfortunately Jekyll requires the ruby dev tools and the windows installer for ruby 2.3.6 doesn't seem to include them. I'm in the process of trying it on cygwin, I'll update when I make any progress.

**Comment by addisonElliott** [comment]
I started looking into this and thought it was a Windows/Cygwin issue. With the default installation, I am able to run the docs without any modifications.

Then I start messing around with your Gemfile and I found the reason:

source 'https://rubygems.org'
gem "minima", "~> 2.0"
gem 'github-pages', group **\<MISSING COLON HERE\>** :jekyll_plugins

So it should be:

source 'https://rubygems.org'
gem "minima", "~> 2.0"
gem 'github-pages', group: :jekyll_plugins

Very easy to miss that!
